portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -132,7 +134,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       String userEmail = UtilityFunctions.getFieldFromJsonObject(jsonObject, "email", "janedoe@gmail.com");
       String currDate = String.valueOf(System.currentTimeMillis());
       long userDate = Long.parseLong(UtilityFunctions.getFieldFromJsonObject(jsonObject, "timestamp", currDate));
-      UtilityFunctions.addToDatastore(userName, userEmail, userDate, userComment, 0, 0, false);
+      UtilityFunctions.addToDatastore(userName, userEmail, userDate, userComment, 0, 0, false, 0, 0);
```

Comment:
For better readability, it is usually useful to add comment on the constant values. Like this:

```
UtilityFunctions.addToDatastore(userName, userEmail, userDate, userComment, /* parentId= */ 0, /* rootId= */ 0, /* isReply= */ false, /* upvotes= */ 0, /* downvotes= */ 0);
```
(also remember to format it so that it does not exceed 100 chars :))

Without these comments, readers need to find the function signature to understand what do these constants mean.
Well-defined variable names (like `userName`) do not need these since readers can reasonably understand them.

This is not in the public style guide, but is a common practice documented in the internal style guide.

Reply:
Made changes in next PR along with line length fix.